src/main/java/com/example/spring/dto/category/CategoryDto.java

```diff
@@ -0,0 +1,9 @@
+package com.example.spring.dto.category;
+
+public record CategoryDto(
+        Long id,
+        String name,
+        String description
+) {

```

Comment:
please add validations in all places where needed

Reply:
here the same